### PR TITLE
tests: relax test-recorder checks

### PR DESCRIPTION
**Pull request by wlozano0collabora**

### CODE REVIEW

File: tests/test-recorder.c [hunk 1]
```diff
@@ -475,7 +475,7 @@ test_recorder_split_bytes (TestFixture * fixture, gconstpointer unused)
 
     if (filenames->next) {
       g_assert_cmpint (labs (stat_buf.st_size - FILE_SEGMENT_LEN_BYTES), <=,
-          FILE_SEGMENT_LEN_BYTES / 5);
+          FILE_SEGMENT_LEN_BYTES / 5 * 1.1);
```
Comment (xhaakon) [inline tests/test-recorder.c]: Perhaps make more obvious where did this `1.1` number come from?

Comment (wlozano0collabora) [reply tests/test-recorder.c]: I see you point, would something like this be enough?

```
// Ratio to apply in file segment length check to ensure proper segment size
// and distribution
#define FILE_SEGMENT_LEN_BYTES_RATIO_LIMIT 1.1
```

Comment (xhaakon) [reply tests/test-recorder.c]: Hm, can we express the allowed size difference as a percentage of `FILE_SEGMENT_LEN_BYTES`? For example `FILE_SEGMENT_LEN_BYTES * 0.25`, meaning the resulting file should differ at most 25% from the expected size? (I thought that number up. Check experimentally which value works!)

Comment (wlozano0collabora) [reply tests/test-recorder.c]: Sure, I just thought that somehow the `/ 5` had some meaning and tried to keep it. I do agree that `* 0.25` looks clearer.

Experiments show that `0.25` is enough so I will push that change.

Comment (justinjoy) [reply tests/test-recorder.c]: Can you add a comment that the ratio is only for testing purposes? Allowing 25% error may be a problem in actual use.

Comment (wlozano0collabora) [reply tests/test-recorder.c]: Sure.

Comment (wlozano0collabora) [reply tests/test-recorder.c]: After some additional testing with `azure` this change seems not to be needed, only in my PC this error may arise with low probability, so I'm dropping it.

**Review by xhaakon — APPROVED** [review]

**Review by xhaakon — APPROVED** [review]

**Review by justinjoy — APPROVED** [review]

**Pull request merged**